service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsLAController.java

```diff
@@ -189,10 +193,14 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBod
                         caseData.getSupportingEvidenceDocumentsTemp(), caseDataBefore.getCorrespondenceDocumentsLA()
                     );
 
+                List<Element<SupportingEvidenceBundle>> sortedDocuments = updatedCorrespondenceDocuments.stream()
```

Comment:
would be good to have method maybe in manageDocumentService that do the sort, to not duplicate this code, and it would be easier test.